docs/reference/aliasing.rst

```diff
@@ -0,0 +1,37 @@
+==================
+Automatic Aliasing
+==================
```

Comment:
Small note, as per Peter's comments, we should avoid using gerunds in the page titles.
https://github.com/canonical/microovn/pull/126#issuecomment-1984172113